examples/clusteredInverse/main.cpp

```diff
@@ -99,60 +95,34 @@ int main(int argc, char *argv[])
 
     // Command Line Parser
     QCommandLineParser parser;
-    parser.setApplicationDescription("Clustered Inverse Example");
+    parser.setApplicationDescription("Start clusteredInverse example");
```

Comment:
Please keep the old more human readable title "Clustered Inverse Example" that is consistent to the other examples